uint8 id_kobuki
uint8 id_task
uint8 id_port
int32 sec
int32 nsec
bool active
